packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -52,20 +56,23 @@ export type VariableValueOptions =
     }
   | SendValuesBaseOptions;
 
+export type TraceReportingOptions<TContext> =
+  | ((
+      request:
+      | GraphQLRequestContextDidResolveOperation<TContext>
+      | GraphQLRequestContextDidEncounterErrors<TContext>,
+    ) => Promise<boolean>)
+  | boolean;
+
 export type GenerateClientInfo<TContext> = (
   requestContext: GraphQLRequestContext<TContext>,
 ) => ClientInfo;
 
 // AS3: Drop support for deprecated `ENGINE_API_KEY`.
-export function getEngineApiKey({
-  engine,
-  skipWarn = false,
-  logger = console,
-}: {
-  engine: EngineReportingOptions<any> | boolean | undefined;
-  skipWarn?: boolean;
-  logger?: Logger;
-}) {
+export function getEngineApiKey(
```

Comment:
This is kinda my fault but I did run prettier on a commit you merged in, and these style-only changes are a result of not taking my changes I guess? If you run prettier on agent.ts they go away. I know prettier is not standard in this repo anymore but the files in this module are mostly prettier-ified and I think we'll be happier continuing to use it for AER. I see a bunch of odd style things in this commit (weird spacing, inconsistent quoting, etc) and I'd rather just not have to mention them and let prettier take care about it.